.github/stale.yml

```
@@ -8,10 +8,9 @@ daysUntilClose: 30
 
 # Issues with these labels will never be considered stale
 exemptLabels:
-  - pinned
-  - security
   - accessibility
-  - "breaking change :boom:"
+  - "breaking changes"
```

double check me, but pretty sure this is the label....

```suggestion
  - "breaking change"
```